Mathlib/FieldTheory/Galois/FiniteDimensional.lean

```diff
@@ -278,6 +278,111 @@ def galoisCoinsertionIntermediateFieldSubgroup [FiniteDimensional F E] [IsGalois
 
 end IsGalois
 
+section
+
+/-In this section we prove that the normal subgroups correspond to the Galois subextensions
+in the Galois correspondence and its related results.-/
+
+variable {K L : Type*} [Field K] [Field L] [Algebra K L]
+
+open IntermediateField
+
+namespace Normal
+
+private lemma of_intermediateField_mem_invariant_under_embedding [Normal K L]
```

Comment:
This lemma can be removed because `of_fixedField_normal_subgroup` can be directly proved using `normal_iff_forall_map_le'`.